src/tyr/isochrone_serializer.cc

```diff
@@ -142,5 +142,66 @@ std::string serializeIsochrones(const Api& request,
 
   return ss.str();
 }
+std::string serializeIsochronePbf(Api& request,
+                                  std::vector<midgard::GriddedData<2>::contour_interval_t>& intervals,
+                                  midgard::GriddedData<2>::contours_t& contours) {
+  // construct pbf output
+  Isochrone& isochrone = *request.mutable_isochrone();
+
+  // construct contours
+  for (size_t isoline_index = 0; isoline_index < contours.size(); ++isoline_index) {
+    const auto& contour = contours[isoline_index];
+    const auto& interval = intervals[isoline_index];
+
+    valhalla::Isochrone::Contour con;
+    con.set_metric(std::get<2>(interval) == "time" ? Isochrone_metric_type_time
+                                                   : Isochrone_metric_type_distance);
```

Comment:
nit: short form is somewhat nicer IMO
```suggestion
    con.set_metric(std::get<2>(interval) == "time" ? Isochrone::time
                                                   : Isochrone::distance);
```